ui/app/src/components/Dialogs/ConfirmDialog.tsx

```diff
@@ -24,18 +23,64 @@ import { defineMessages, useIntl } from 'react-intl';
 import StandardAction from '../../models/StandardAction';
 import Dialog from '@material-ui/core/Dialog';
 import { useUnmount } from '../../utils/hooks';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import confirmGraphicUrl from '../../assets/confirm.svg';
 
 const messages = defineMessages({
-  ok: {
-    id: 'words.ok',
-    defaultMessage: 'Ok'
+  accept: {
+    id: 'words.accept',
+    defaultMessage: 'Accept'
   },
   cancel: {
     id: 'words.cancel',
     defaultMessage: 'Cancel'
   }
 });
 
+const confirmDialogStyles = makeStyles(() =>
+  createStyles({
+    dialog: {
+      '& .MuiPaper-root': {
+        maxWidth: '350px',
```

Comment:
Can't this be achieved with the dialog built in size prop? What's the min? If it's close, is enough.

Reply:
@rart when setting maxWidth prop to 'xs', its max-width is set to 444px, please let me know if it's a valid width, or if we leave it in 350px

Reply:
Good enough. 👍🏻